SourceKitten.xcworkspace/xcshareddata/SourceKitten.xcscmblueprint

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
For reference, this was added to the `Swift.gitignore` (https://github.com/github/gitignore/pull/1547), but not based on any documentation though.